src/test/java/DukeParserTest.java

```diff
@@ -0,0 +1,122 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import org.junit.jupiter.api.Test;
+
+import duke.command.Parser;
+import duke.error.DukeException;
+import duke.task.DeadlineTask;
+import duke.task.EventTask;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+
+
+public class DukeParserTest {
+    @Test
+    public void createTaskCommand() {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/M/yyyy");
+        Parser parser = new Parser();
+        Ui p = new Ui();
```

Comment:
Even as a temporary variable, i feel that `p` has no link with the `Ui` class. Consider changing it to `ui`. This suggestion applies to other test cases that follow a similar naming.
```suggestion
        Ui ui = new Ui();
```